### PR TITLE
Installation Geonode 2.6, new style and taxtweb

### PR DESCRIPTION
**Pull request by hascar**
Installation Geonode, apply modify for new style and installation taxtweb with relative tests with moon

### CODE REVIEW

File: .gem_ffox_init.sh [hunk 1]
```diff
@@ -0,0 +1,35 @@
+export GEM_FIREFOX_ON_HOLD=
```
Comment (nastasi-oq) [inline .gem_ffox_init.sh]: With this line any value assigned in the CI job will be ignored, remove it, please.

**Comment by nastasi-oq** [comment]
Please remove all 'geonode' images from ``geonode/geonode`` folder.

File: .gem_ffox_init.sh (hunk 1 above)
Comment (hascar) [inline .gem_ffox_init.sh]: Deleted here https://github.com/gem/oq-platform2/pull/1/commits/34a1d13ef05a8a4619856617693852a495a19c01

File: verifier.sh [hunk 2]
```diff
@@ -0,0 +1,386 @@
...
+    ret=$1
+
+    echo
+    echo "USAGE:"
+    echo "    $0 devtest <branch-name> [<plugins-branch-name>]"
+    echo "                                                 put oq-platform sources in a lxc,"
+    echo "                                                 setup environment and run development tests."
+    echo "    $0 devtest <branch-name> <smtp_address> ['notest']"
+    echo "                                                 production installation and tests."
+    echo
```
Comment (nastasi-oq) [inline verifier.sh]: Fix ``usage`` function.

File: verifier.sh [hunk 3]
```diff
@@ -0,0 +1,386 @@
...
+#  _devtest_innervm_run <branch_id> <lxc_ip> - part of source test performed on lxc
+#                     the following activities are performed:
+#                       files and install them
+#
+#      <branch_id>    name of the tested branch
+#      <lxc_ip>       the IP address of lxc instance
+#
+_devtest_innervm_run () {
+    local i old_ifs pkgs_list dep git_branch="$1" branch_geonode="$2"
+
```
Comment (nastasi-oq) [inline verifier.sh]: notests not assigned from $3 but still used.

File: verifier.sh [hunk 4]
```diff
@@ -0,0 +1,386 @@
...
+    scp "${lxc_ip}:dev_*.png" "out/" || true
+    scp "${lxc_ip}:xunit-platform-dev.xml" "out/" || true
+}
+
+
+#
+#  sig_hand - manages cleanup if the build is aborted
+#
+sig_hand () {
+
```
Comment (nastasi-oq) [inline verifier.sh]: empty line can be removed.

File: pavement.py [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+from paver.easy import task, options, cmdopts, needs
+from paver.easy import path, sh, info, call_task
+from paver.easy import BuildFailure
+
+
+GEM_LOCAL_SETTINGS_TMPL = 'local_settings.py.tmpl'
+
+
+def _write_local_settings(lxc_ip,webuiurl):
```
Comment (nastasi-oq) [inline pavement.py]: a space before webuiurl is needed.

File: pavement.py [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+@task
+@cmdopts([
+    ('lxc_ip=', 'l', 'Bind server to provided IP address and port number.'),
+    ('webuiurl=', 'u', 'Bind server to provided URL of webui.')
+])
+def setup():
+    lxc_ip = options.get('lxc_ip', '')
+    webuiurl = options.get('webuiurl', '')
+    # info(lxc_ip)
+    _write_local_settings(lxc_ip,webuiurl)
```
Comment (nastasi-oq) [inline pavement.py]: a space before webuiurl is needed.

File: verifier-guest.sh [hunk 7]
```diff
@@ -0,0 +1,94 @@
...
+cd ~
+# Create a symbolic link in your virtualenv
+ln -s /usr/lib/python2.7/dist-packages/osgeo env/lib/python2.7/site-packages/osgeo
+
+sudo cp $HOME/"$GIT_REPO"/urls.py $HOME/geonode/geonode
+
+## clone and setting pythonpath taxtweb and oq-platform2
+cd ~
+git clone https://github.com/gem/oq-platform-taxtweb.git
+export PYTHONPATH=:~/oq-platform2:~/oq-platform-taxtweb
```
Comment (nastasi-oq) [inline verifier-guest.sh]: the first ``:`` has no meaning and maybe is better use ``$HOME`` instead of ``~``.

File: verifier.sh [hunk 8]
```diff
@@ -0,0 +1,386 @@
...
+}
+
+#
+#  _devtest_innervm_run <branch_id> <lxc_ip> - part of source test performed on lxc
+#                     the following activities are performed:
+#                       files and install them
+#
+#      <branch_id>    name of the tested branch
+#      <lxc_ip>       the IP address of lxc instance
+#
```
Comment (nastasi-oq) [inline verifier.sh]: Please update function description

File: verifier.sh [hunk 9]
```diff
@@ -0,0 +1,386 @@
...
+
+    return 0
+}
+
+#
+#  devtest_run <branch_id> <branch_geonode> - main function for development test
+#      <branch_id>        name of the tested branch
+#      <branch_geonode>   name of the geonode branch
+#
+devtest_run () {
```
Comment (nastasi-oq) [inline verifier.sh]: Please update function description

File: pavement.py (hunk 5 above)
Comment (hascar) [inline pavement.py]: Correctly here https://github.com/gem/oq-platform2/pull/1/commits/3a77c98c02ee646d3982153eee30f96f83577e7e

File: pavement.py (hunk 6 above)
Comment (hascar) [inline pavement.py]: Correctly here https://github.com/gem/oq-platform2/pull/1/commits/3a77c98c02ee646d3982153eee30f96f83577e7e

File: verifier.sh (hunk 8 above)
Comment (hascar) [inline verifier.sh]: Correctly here https://github.com/gem/oq-platform2/pull/1/commits/3e7f8a02491e0ea9e81b1c73fae58d042c693a3d

File: verifier.sh (hunk 9 above)
Comment (hascar) [inline verifier.sh]: Correctly here https://github.com/gem/oq-platform2/pull/1/commits/3e7f8a02491e0ea9e81b1c73fae58d042c693a3d

File: verifier.sh (hunk 3 above)
Comment (hascar) [inline verifier.sh]: Correctly here https://github.com/gem/oq-platform2/pull/1/commits/4b1ca1ad5f18a8005e5ff5f00ce248bbc7663a6c

File: verifier.sh (hunk 2 above)
Comment (hascar) [inline verifier.sh]: Correctly here https://github.com/gem/oq-platform2/pull/1/commits/4b924d44ff592a31eedaf5f6f10fcc305961ca19

File: verifier.sh [hunk 10]
```diff
@@ -36,8 +36,10 @@ usage () {
 
     echo
     echo "USAGE:"
-    echo "$0 devtest <branch-name> and if not running test use <notests>"
-    echo "development  installation and tests."
+    echo "    $0 devtest <branch-name> ['notests']"
+    echo "                                     development installation and tests."
+    echo "         <branch-name>    name of branch repo"
+    echo "         ['notests']    if not running the tests"
```
Comment (nastasi-oq) [inline verifier.sh]: [] are not needed here

**Pull request merged**